utils/snowplow_split_events_model_gen.py

```diff
@@ -412,20 +246,20 @@ def validate_json(jsonData: dict, schema: dict = None, validate: bool = True) ->
 select
     event_id
     , collector_tstamp
```

Comment:
You might need to add the `date(collector_tstamp) as collector_tstamp_date` line if the `target.type == 'Databricks'`